tech-project-back-end/Models/DetailOrder.cs

```diff
@@ -7,19 +7,19 @@ public class DetailOrder
     {
         [Key]
         [Column("order_id", TypeName = "varchar(36)")]
-        public string order_id { get; set; }
+        public string Order_id { get; set; }
 
         [Key]
         [Column("product_id", TypeName = "varchar(36)")]
-        public string product_id { get; set; }
+        public string Product_id { get; set; }
 
         [Column("price", TypeName = "bigint(20) UNSIGNED")]
-        public long price { get; set; }
+        public long Price { get; set; }
 
-        [Column("quality", TypeName = "int(11)")]
-        public int quality { get; set; }
+        [Column("quantity", TypeName = "int(11)")]
+        public int Quantity { get; set; }
 
-        [ForeignKey("order_id")]
+        [ForeignKey("Order_id")]
         public virtual Order Order { get; set; }
 
```

Comment:
rename all the variable in models into pascal case but keep the column name as (abc_xyz)